ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1223,6 +1234,10 @@ def test_method(
         else:
             ret_device = None
 
+        assert ivy.nested_map(
+            ret, lambda x: ivy.is_ivy_array(x) if ivy.is_array(x) else True
```

Comment:
`ivy.is_ivy_array()` would be bound to global ivy, which has it's backend unchanged, we should use local Ivy.